modules/geo-layers/src/tile-layer/tile-2d-header.js

```diff
@@ -17,12 +18,22 @@ export default class Tile2DHeader {
 
     this.onTileLoad = onTileLoad;
     this.onTileError = onTileError;
+
+    this.transformToWorld = null;
   }
 
   get data() {
     return this._isLoaded ? this.content : this._loader;
   }
 
+  get dataInWorldCoordinates() {
+    if (this.transformToWorld) {
+      return this.transformToWorld(this, this.content);
```

Comment:
- This is still recalculating all the transforms every time this function is called. Content and transformToWorld are not expected to change.
- Not handling the case when content is still loading.